test/e2e/node/runtimeclass.go

```diff
@@ -143,18 +138,61 @@ var _ = SIGDescribe("RuntimeClass [Feature:RuntimeClass]", func() {
 	// TODO(tallclair): Test an actual configured non-default runtimeHandler.
 })
 
+// createCRD creates CRD for RuntimeClass.
+func createCRD(f *framework.Framework) {
+	crdFilePath := "cluster/addons/runtimeclass/runtimeclass_crd.yaml"
+	data := testfiles.ReadOrDie(crdFilePath, Fail)
+	var crd apiextensionsv1beta1.CustomResourceDefinition
+	framework.ExpectNoError(runtime.DecodeInto(legacyscheme.Codecs.UniversalDecoder(), data, &crd),
+		"failed to decode the content of the CRD file")
+
+	crds := f.APIExtensionsClientSet.ApiextensionsV1beta1().CustomResourceDefinitions()
+	By("Creating CRD for RuntimeClass")
+	_, err := crds.Create(&crd)
+	if err != nil {
+		if errors.IsAlreadyExists(err) {
+			By("The RuntimeClass CRD is already created")
+		} else {
+			framework.ExpectNoError(err, "Failed to create %s CRD", crd.Name)
+		}
+	}
+
+	var crdUID string
+	By("Waiting up to 5 minutes for the RuntimeClass CRD to be established")
+	framework.ExpectNoError(wait.PollImmediate(
+		500*time.Millisecond, 5*time.Minute, func() (bool, error) {
+		crd, err := crds.Get(crd.Name, metav1.GetOptions{})
+		if err != nil {
+			return false, err
+		}
+		for _, cond := range crd.Status.Conditions {
+			switch cond.Type {
+			case apiextensionsv1beta1.Established:
+				if cond.Status == apiextensionsv1beta1.ConditionTrue {
+					crdUID = string(crd.GetUID())
+					return true, nil
+				}
+			}
+		}
+		return false, nil
+	}))
+	By(fmt.Sprintf("The RuntimeClass CRD (%s) is established", crdUID))
+}
+
 // createRuntimeClass generates a RuntimeClass with the desired handler and a "namespaced" name,
 // synchronously creates it, and returns the generated name.
 func createRuntimeClass(f *framework.Framework, name, handler string) string {
 	uniqueName := fmt.Sprintf("%s-%s", f.Namespace.Name, name)
 	rc := runtimeclasstest.NewRuntimeClass(uniqueName, handler)
 	rc, err := f.NodeAPIClientSet.NodeV1alpha1().RuntimeClasses().Create(rc)
 	framework.ExpectNoError(err, "failed to create RuntimeClass resource")
+	By(fmt.Sprintf("RuntimeClass %s is created", uniqueName))
 	return rc.GetName()
 }
 
 // createRuntimeClass creates a test pod with the given runtimeClassName.
 func createRuntimeClassPod(f *framework.Framework, runtimeClassName string) *v1.Pod {
+	By("Creating RuntimeClassPod")
```

Comment:
nit: include the runtimeClassName:
```suggestion
	By("Creating Pod using RuntimeClass %s", runtimeClassName)
```

Reply:
Done